### PR TITLE
Prep community.aws 7.0.0 release

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Prep community.aws 7.0.0 release

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

several

### CODE REVIEW

**Comment by alinabuzachis** [comment]
Close and re-open to trigger github workflows.

**Comment by alinabuzachis** [comment]
It is failing because this one has not been merged yet https://github.com/ansible-collections/amazon.aws/pull/1837

**Comment by alinabuzachis** [comment]
@tremble do we want to wait until https://github.com/ansible-collections/amazon.aws/pull/1837 is merged or keep amazon.aws: '>=7.0.0-dev0' as dependency?

File: galaxy.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+version: 7.0.0
 readme: README.md
 authors:
   - Ansible (https://github.com/ansible)
 description: A variety of Ansible content to help automate the management of AWS services.
 license_file: COPYING
 tags: [community, aws, cloud, amazon]
 dependencies:
-  amazon.aws: '>=7.0.0-dev0'
+  amazon.aws: '>=8.0.0-dev0'
```
Comment (tremble) [inline galaxy.yml:12]:
```suggestion
  amazon.aws: '>=7.0.0'
```

This is still the stable-7 branch, we only need amazon.aws 7.x :)

`>= 8.0.0-dev0` will be needed on the `main` branch, but can't be done without https://github.com/ansible-collections/amazon.aws/pull/1837

**Review by tremble — CHANGES_REQUESTED** [review]

**Review by tremble — APPROVED** [review]

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Pull request merged**